nengo_viz/static/viz_netgraph.js

```diff
@@ -104,6 +131,7 @@ VIZ.NetGraph = function(parent, args) {
             self.offsetY = (self.offsetY + yy) / scale - yy;
             
             self.scale = scale * self.scale;
+            interact.margin(30*self.scale);
```

Comment:
Not sure, whether I agree, that we do not need a corner drag for ensembles. Even though the aspect ratio is fixed, the behaviour for a corner drag is slightly different from a side drag.